test/unit/helpers/ConfigHelper.ts

```diff
@@ -278,8 +290,15 @@ test(`promptOptions 10 - autoRegister backwards compatibility for custom integra
       autoPrompt: false,
     },
     slidedown: {
-      enabled: true,
-      autoPrompt: false,
+      prompts: [{
```

Comment:
Should keep the old format so we are still testing what could be in the wild as is.

Can use `getPromptOptionsFromFinalAppConfig` to make the tests pass.